docs/en/deploy/Running-Alluxio-Using-Vagrant.md

```diff
@@ -5,61 +5,120 @@ nickname: Alluxio on Vagrant
 group: Deploying Alluxio
 priority: 5
 ---
-TODO(david): Outline
-  - Table of Contents
-  - Introduction
-  - Prerequisite for using Vagrant
-  - Deploy using Vagrant on Virtual Box
-    - Launch Cluster
-    - Access / Verify the Cluster
-    - Destroy 
-  - Deploy using Vagrant on EC2
-    - Launch Cluster
-    - Access / Verify the Cluster
-    - Destroy
-    - Advanced Tips
-      -Spot Instance
-  - Deploy using Vagrant on GCE
-    - Launch Cluster
-    - Access / Verify the Cluster
-    - Destroy
-
-TODO(david): combine content from Deploy on EC2, GCE, Virtual Box using Vagrant 
-
 * Table of Contents
 {:toc}
 
-Alluxio can be deployed on Amazon EC2 using the [Vagrant scripts](https://github.com/alluxio/alluxio/tree/master/deploy/vagrant) that come with Alluxio. The scripts let you create, configure, and destroy clusters that come automatically configured with [Amazon S3](https://s3.amazonaws.com/).
+## Introduction
+Alluxio can be deployed locally or in the cloud using the [Vagrant scripts](https://github.com/alluxio/alluxio/tree/master/deploy/vagrant) that come with Alluxio. The scripts let you create, configure, and destroy clusters that come automatically configured locally, with [Amazon EC2](https://ec2.amazon.com/) or with [Google Compute Engine](https:/cloud.google.com).
 
-## Prerequisites
+## Common Prerequisites
+There are several prerequisites for all three deployment scenarios that are suppported. 
 
-**Install Vagrant and the AWS plugins**
+* Download and Install [Vagrant](https://www.vagrantup.com/downloads.html)
+* [Clone Alluxio Repository](https://github.com/Alluxio/alluxio) to your local machine
+* Download and Install [python>=2.7](https://www.python.org/), not python3.
+* Install python library dependencies, under `deploy/vagrant` directory in your home directory, run:
 
-Download [Vagrant](https://www.vagrantup.com/downloads.html)
+```bash
+$ sudo bash bin/install.sh
+```
 
-Install AWS Vagrant plugin:
+Alternatively, you can manually install [pip](https://pip.pypa.io/en/latest/installing/), and then
+in `deploy/vagrant` run:
 
-{% include Running-Alluxio-on-EC2/install-aws-vagrant-plugin.md %}
+```bash
+$ sudo pip install -r pip-req.txt
+```
 
-**Install Alluxio**
 
-[Clone Alluxio Repository](https://github.com/Alluxio/alluxio) to your local machine.
+## Deploy locally on Virtual Box using Vagrant 
```

Comment:
not sure we need "using Vagrant", all three sections in this doc require Vagrant in this doc, right?
it seems not adding any information